tests/test_vms_exist.py

```diff
@@ -7,6 +7,9 @@
 
 DEBIAN_VERSION = "bookworm"
 
+with open("config.json") as f:
```

Comment:
Note to self: this is a little duplicative of the `self.config` but otherwise it can't be used in the skipIf().